rust/protocol/src/sealed_sender.rs

```diff
@@ -1060,6 +1343,11 @@ pub fn sealed_sender_multi_recipient_fan_out(data: &[u8]) -> Result<Vec<Vec<u8>>
     Ok(messages)
 }
 
+/// Decrypt a sealed-sender message in either the v1 or v2 format.
+///
+/// This method produces an [UnidentifiedSenderMessageContent] with
+/// the plaintext information about the sender's identity, which is then used to finally decrypt the
+/// underlying encrypted message in [sealed_sender_decrypt].
```

Comment:
I have clarified that this method extracts the sender's identity from the underlying message and removed the extraneous mention of `[UnidentifiedSenderMessageContent]`, since that is in the return type of the method.